src/preferences/dialog/dlgprefeffects.cpp

```diff
@@ -15,6 +15,16 @@ DlgPrefEffects::DlgPrefEffects(QWidget* pParent,
     m_pAvailableEffectsModel = new EffectSettingsModel();
 
     m_pAvailableEffectsModel->resetFromEffectManager(pEffectsManager);
+    for (auto profile : m_pAvailableEffectsModel->profiles()) {
+        EffectManifestPointer pManifest = profile->pManifest;
+
+        // Blacklisted Non-native effects by default
+        bool defaultValue = QString::localeAwareCompare(pManifest->backendName(), tr("Native")) == 0;
```

Comment:
I think this might fail if the translator accidentally translates the string "Native" here differently than in nativebackend.cpp. Instead of storing a user-facing string on the manifest to identify the effect type, let's add a new enum class to src/effects/defs.h and store that with the manifest:
```
enum class EffectType {
  BuiltIn,
  LV2
}
```
I'm also unsure the word "Native" is very clear. I think "Built-in" more accurately describes the difference versus external effect plugins.

Reply:
Yes, "Built-in" seems to be more clear. Should we change every occurrence of "Native" to "Built-in" in the codebase? I think we can start a topic on zulip for the same.

Reply:
Let's go ahead and rename the classes and source code files that use "native" too. It would be confusing for developers to use a different term in the UI compared to the code.

Reply:
Should I make those changes in this PR only? Can it not a create a merge conflict? I think renaming the native directory and all instances of "Native" should be done in a separate PR.

Reply:
We are almost done with this PR, so I think it is okay to do it in this one. Usually I would suggest doing it in another PR though.